### PR TITLE
[TW-582] doc scheduling collection runs

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ warning: false
 
 ---
 
-The _Collection Runner_ enables you to run the API requests in a collection in a specified sequence. It logs your request [test results](/docs/writing-scripts/test-scripts/) and can use [scripts](/docs/writing-scripts/intro-to-scripts/) to pass data between requests and alter the request workflow.
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/intro-to-collection-runs.md:39]: Up in front matter, make sure to change the date.

File: src/pages/docs/running-collections/running-collections-overview.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Running collections - overview"
+updated: 2021-02-16
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/running-collections-overview.md:3]: change the date here

File: src/pages/docs/running-collections/running-collections-overview.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Running collections - overview"
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/running-collections-overview.md:2]:
```suggestion
title: "Running collections overview"
```

Comment (jkonrath-postman) [reply src/pages/docs/running-collections/running-collections-overview.md:2]: This mirrors the naming on other overview topics.

File: src/pages/docs/running-collections/scheduling-collection-runs-monitors.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Scheduling collection runs with monitors"
+updated: 2021-02-16
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs-monitors.md:3]: Change date

File: src/pages/docs/running-collections/scheduling-collection-runs-monitors.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  - type: dynamic_blog
+    name: "Blog Posts"
+    blog_tag: "monitors"
+
+warning: false
+---
+
+You can automate [collection runs](/docs/running-collections/intro-to-collection-runs/) using [monitors](/docs/monitoring-your-api/intro-monitors/) to schedule runs and receive reports on your request test results.
+
+<!--TODO: Add language that says the scheduled collection run method is preferable. Key off marketing language or a blog post.-->
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs-monitors.md:26]: @malvikach let us know what language you want here. Should we be saying running with monitors is deprecated? That collection runner is preferred?

File: src/pages/docs/running-collections/scheduling-collection-runs-monitors.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+> Starting with version 10, you can also schedule collection runs with the [**Collection Runner**](/docs/running-collections/scheduling-collection-runs/).
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs-monitors.md:27]:
```suggestion
> Starting with Postman v10, you can also schedule collection runs with the [**Collection Runner**](/docs/running-collections/scheduling-collection-runs/).
```

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 7]
```diff
@@ -1,8 +1,6 @@
 ---
-title: "Scheduling runs with monitors"
-order: 56.1
+title: "Scheduling collection runs"
 updated: 2021-02-16
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:3]: update date

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 8]
```diff
@@ -23,20 +21,127 @@ contextual_links:
...
+When you schedule a collection run with the Collection Runner, the scheduled run is added to the collection's **Runs** tab. You can view, pause, edit, and delete scheduled collection runs from the collection's **Runs** tab. These scheduled runs execute in the Postman Cloud.
 
-<img alt="Monitors v8" src="https://assets.postman.com/postman-docs/create-a-monitor-v8.jpg" height="400px"/>
+<img alt="Runs tab v10" src="https://assets.postman.com/postman-docs/v10/scheduled-runs-v10.jpg"/>
 
-Any monitors already attached to your collection will appear. Select __Create a monitor__ to add one (or __+ Create a new Monitor__ if you already have one on the collection).
+> * Schedules share permissions with their collections. For example, if you have permissions to edit a collection, you'll you'll be able to edit that collection's schedules.
+> * Scheduled collection runs support [custom run orders](/docs/running-collections/building-workflows/).
+> * Only personal, private, and team workspaces support scheduling collection runs.
+> * Scheduled runs are executed in the Postman Cloud. Use of the Postman Cloud is limited by your Postman account.
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:33]: Not sure what that second sentence means. Is that a pricing thing? Where can people learn more? (@malvikach ?)

**Review by jkonrath-postman — COMMENTED** [review]
A few minor things - let me know when we're caught up on UX drift, and I can do another pass and approve.

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 9]
```diff
@@ -23,20 +21,127 @@ contextual_links:
 warning: false
 ---
 
-You can automate [collection runs](/docs/running-collections/intro-to-collection-runs/) using [monitors](/docs/monitoring-your-api/intro-monitors/) to schedule runs and receive reports on your request test results.
+Many Postman users run collections at specific times each day or week. For example, you may want to test your API's endpoints every week to observe availability, performance, or reliability. You can schedule collection runs using the Collection Runner. You can also schedule collection runs using [monitors](/docs/running-collections/scheduling-collection-runs-monitors/).
```
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:24]: @malvikach what are pros/cons to scheduling collection runs with the collection runner vs. monitors? Are there use cases for each?

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 10]
```diff
@@ -23,20 +21,127 @@ contextual_links:
...
+    * How many times you want the collection to run (iterations)
+    * A `JSON` or `CSV` [data file](/docs/running-collections/working-with-data-files/) (optional)
+    * Notification recipients (optional)
+    * Advanced settings (optional)
+      * Retry if run fails
+      * Set request timeout
+      * Set delay between requests
+      * Follow redirects
+      * Enable SSL validation
+
```
Comment (mark-anthony-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:74]: Might be good to add an optional step about reordering the requests. Here's the text you have on the "Using the Collection Runner" page:

By default, your requests run in the sequence they're listed in the collection. If you need to change the order of execution, select and drag a request to its new location in the order. You can also remove an individual request from the run by clearing the checkbox next to its name.

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 11]
```diff
@@ -23,20 +21,127 @@ contextual_links:
...
+1. Select **Collections** in the sidebar and select the collection with the scheduled run you want to view.
+
+1. Select the **Runs** tab.
+
+    ![Scheduled runs tab v10](https://assets.postman.com/postman-docs/v10/scheduled-runs-tab-v10.jpg)
+
+1. Select the **Scheduled runs** tab. A list of your scheduled runs appears.
+
+1. Select **View** next to your scheduled run.
```
Comment (mark-anthony-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:87]: I think you need to add more content here to discuss the view scheduled run page, including a screenshot. You see a history of your runs. You can hover over a bar for more information, or select a bar for details. There are dropdowns you can use to filter which requests are displayed. And there are buttons to run the collection or refresh the results. For reference see: https://learning.postman.com/docs/monitoring-your-api/viewing-monitor-results/

**Review by mark-anthony-postman — COMMENTED** [review]
These are some great changes. I enjoyed all the clearly written steps! 💎✨

Made some comments, mostly just small stuff. My one big comment is that I think you need to add content about the View Scheduled Run page (what you see after you select "View" next to a scheduled run). It's similar to viewing monitor results: https://learning.postman.com/docs/monitoring-your-api/viewing-monitor-results/

File: src/pages/docs/running-collections/scheduling-collection-runs-monitors.md (hunk 4 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs-monitors.md:3]: Changed to 2022-10-19

File: src/pages/docs/running-collections/running-collections-overview.md (hunk 2 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/running-collections-overview.md:3]: Changed to 2022-10-19

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 7 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:3]: Changed to 2022-10-19

File: src/pages/docs/running-collections/running-collections-overview.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+    name: "API Monitoring | The Exploratory"
+    url: "https://youtu.be/tDQzY1Hn2LY"
+  - type: dynamic_blog
+    name: "Blog Posts"
+    blog_tag: "monitors"
+
+warning: false
+---
+
+Postman enables you to run collections in several ways. You can:
```
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/running-collections-overview.md:24]: Added, thanks Mark.

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 13]
```diff
@@ -23,20 +21,127 @@ contextual_links:
...
 ---
 
-You can automate [collection runs](/docs/running-collections/intro-to-collection-runs/) using [monitors](/docs/monitoring-your-api/intro-monitors/) to schedule runs and receive reports on your request test results.
+Many Postman users run collections at specific times each day or week. For example, you may want to test your API's endpoints every week to observe availability, performance, or reliability. You can schedule collection runs using the Collection Runner. You can also schedule collection runs using [monitors](/docs/running-collections/scheduling-collection-runs-monitors/).
 
-To add or access monitors for a particular collection, select __Monitors__ in the sidebar.
+When you schedule a collection run with the Collection Runner, the scheduled run is added to the collection's **Runs** tab. You can view, pause, edit, and delete scheduled collection runs from the collection's **Runs** tab. These scheduled runs execute in the Postman Cloud.
 
-<img alt="Monitors v8" src="https://assets.postman.com/postman-docs/create-a-monitor-v8.jpg" height="400px"/>
+<img alt="Runs tab v10" src="https://assets.postman.com/postman-docs/v10/scheduled-runs-v10.jpg"/>
```
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:28]: Fixed

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 14]
```diff
@@ -23,20 +21,127 @@ contextual_links:
...
+
+## Viewing your scheduled run
+
+1. Select **Collections** in the sidebar and select the collection with the scheduled run you want to view.
+
+1. Select the **Runs** tab.
+
+    ![Scheduled runs tab v10](https://assets.postman.com/postman-docs/v10/scheduled-runs-tab-v10.jpg)
+
+1. Select the **Scheduled runs** tab. A list of your scheduled runs appears.
```
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:85]: Fixed

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:24]: Thanks @godfrzero and @mark-anthony-postman for the feedback. I've added some more info about which method is preferred for each use case, but I need more details about *why* they're preferred for each. Maybe @malvikach can provide some direction.

[the remark this answers is not in the document]
Comment (godfrzero) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:24]: As it stand right now:

- A monitor is preferred if the user intends to set up alerting and other integrations in addition to the run. For example, if the run is meant to trigger on-call upon failure.
- For all other cases, like when transitioning from the collection runner, a scheduled collection run will do.

[the remark this answers is not in the document]
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:24]: Added a note to `scheduling-collection-runs-monitors.md` and `scheduling-collection-runs.md` summarizing the above distinction.

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 10 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:74]: Added

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 8 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:33]: Removed the "Use of the Postman cloud..." bit and added a new bullet `Scheduled collection runs have the same [usage limits as monitors](/docs/monitoring-your-api/monitor-usage/)`

File: src/pages/docs/running-collections/intro-to-collection-runs.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/intro-to-collection-runs.md:39]: Fixed

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 11 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md]: Great suggestion, thank you. I added a little more content to the procedure plus a link to a new doc covering the various controls. I copied and repurposed the Monitors version of the same page.

File: src/pages/docs/running-collections/scheduling-collection-runs-monitors.md (hunk 5 above)
Comment (malvikach) [reply src/pages/docs/running-collections/scheduling-collection-runs-monitors.md]: @jkonrath-postman @petewitcher Running collections with monitors are not deprecated. We have to be subtle about placing scheduled collection runs such that it implicitly suggests to user that that's preferred. We should not be doing any explicit call outs that use x vs use y. That makes it a bit more prescriptive and currently we are not taking an explicit stance on this. 
As our product evolves, we will start to make this call outs at some point.

Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs-monitors.md]: Thanks @malvikach I'll remove the comment and resolve this conversation. Note there is language in the docs that says scheduled collection runs are useful for automating testing and monitors are useful for checking performance, availability, and reliability. Let me know if that language should change.

File: src/pages/docs/running-collections/running-collections-overview.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Running collections - overview"
+updated: 2022-10-19
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/running-collections-overview.md:3]: Make sure to flip all the dates to 2022-11-09

File: src/pages/docs/running-collections/viewing-scheduled-collection-runs.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+    name: "Prerequisites"
+  - type: link
+    name: "Using the Collection Runner"
+    url: "/docs/running-collections/intro-to-collection-runs/"
+
+warning: false
+
+---
+
+Many Postman users run collections at specific times each day or week. For example, you may want to run a collection that tests your API every day. You can use the Collection Runner to [schedule collection runs](/docs/running-collections/scheduling-collection-runs/) to execute automatically at specified dates and times.
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/viewing-scheduled-collection-runs.md:15]:
```suggestion
You can run collections at specific times each day or week. For example, you may want to run a collection that tests your API every day. You can use the Collection Runner to [schedule collection runs](/docs/running-collections/scheduling-collection-runs/) to execute automatically at specified dates and times.
```

**Review by jkonrath-postman — APPROVED** [review]
A few comments, but otherwise LGTM

File: src/pages/docs/running-collections/running-collections-overview.md (hunk 15 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/running-collections-overview.md:3]: Done

**Pull request merged**